### ISSUE TITLE
Group notification settings - Vibration setting is ignored

### ISSUE THREAD

**Issue by error401de** [signalapp/Signal-Android#13561]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Vibration is disabled in customised group notification settings. Phone still vibrates on incoming messages.

### Steps to reproduce
Set up as in screenshots below

**Actual result:** Vibration
**Expected result:** No vibration

### Screenshots
<img width="293" alt="image" src="https://github.com/signalapp/Signal-Android/assets/34032466/e7136326-94a6-4b9c-b148-119cd58762c4">

<img width="296" alt="image" src="https://github.com/signalapp/Signal-Android/assets/34032466/3508fb70-caf3-439a-8121-451dd430623d">

### Device info
**Device:** Samsung Galaxy S24
**Android version:** 14
**Signal version:** 7.6.2

Does anybody know it this can be fixed?

Thanks

**Comment by nicholas-signal** [comment]
Hello, please provide a debug log. Thanks!

**Comment by lukasgraesslin** [comment]
Can confirm exactly the same behaviour with 7.8.1 on Android 13.

**Comment by error401de** [comment]
@nicholas-signal As I'm not the only one: Can you please provide a mail to send the logs or replicate the issue? It might be a small thing, but if you're in a lot of groups as I am, it's really intrusive. Thanks!

**Comment by nicholas-signal** [comment]
You may send the URL to support@signal.org and please reference this github issue number (#13561) in the email. Thanks!

**Comment by error401de** [comment]
@nicholas-signal Done, thanks!

**Comment by error401de** [comment]
Hi @nicholas-signal,
is there anything you can share? 
Thanks!

**Comment by error401de** [comment]
This issue is still ongoing.

**Comment by error401de** [comment]
Guys...

**Comment by error401de** [comment]
It's one year open now... Any statement on this @greyson-signal ?

**Comment by cody-signal** [comment]
Yeah, I've finally be able to reproduce it. We'll look into it sorry this one floated so long.

**Comment by cody-signal** [comment]
Based on my testing this seems to be some sort of expected but not user expected behavior. If I disable vibrate and have my phone set to silent (not vibrate) or notification sound enabled, it doesn't vibrate. If I switch my phone to vibrate, then it will vibrate but at a lower intensity. If I set my notification sound to be none and stay in vibrate, it doesn't vibrate.

So it seems like because it has a sound associated with it still, and it can't play the sound if set to vibrate, it vibrates. No sound and no vibrate, it won't vibrate. Does that make sense? No not really imo, but is what it is. I'm curious if you see something similar on your device/android OS version.

**Comment by error401de** [comment]
I can indeed replicate the behaviour. To me it actually fixes the issue, as I also don't need the sound. So thanks for sharing the observation!
It would be interesting to see a comparison with similar messengers, unfortunately i don't have another one I could test with.
